3.0/README.md

```diff
@@ -39,28 +39,76 @@ Implementations MUST treat unknown keys as if they weren't present. However, imp
 
 ## 3.1 `attribution`
 
-REQUIRED|OPTIONAL (description of dependents/dependencies)
+OPTIONAL. Default: null.
 
-**Description:**
+Contains an attribution to be displayed when the map is shown to a user. Implementations MAY decide to treat this as HTML or literal text. For security reasons, make absolutely sure that this field can't be abused as a vector for XSS or beacon tracking.
 
-**Example:**
+```JSON
+{
+  "attribution": "<a href='http://openstreetmap.org'>OSM contributors</a>"
+}
+```
 
 ## 3.2 `bounds`
 ## 3.3 `center`
 ## 3.4 `data`
 ## 3.5 `description`
+
+OPTIONAL. Default: null.
+
+A text description of the tileset. The description can contain any legal character. Implementations SHOULD NOT interpret the description as HTML.
+
+```JSON
+{
+  "description": "We are volcanoes. When we women offer our experience as our truth, as human truth, all the maps change. There are new mountains."
+}
+```
+
 ## 3.6 `grids`
 ## 3.7 `legend`
 ## 3.8 `maxzoom`
 ## 3.9 `minzoom`
+
 ## 3.10 `name`
+
+OPTIONAL. Default: null.
+
+A name describing the tileset. The name can contain any legal character. Implementations SHOULD NOT interpret the name as HTML.
+
+```JSON
+{
+  "name": "Earthsea v2"
+}
+```
+
 ## 3.11 `scheme`
 ## 3.12 `template`
 ## 3.13 `tilejson`
+
+REQUIRED.
+
+A semver.org style version number. Describes the version of the TileJSON spec that is implemented by this JSON object.
+
+```JSON
+{
+  "tilejson": "3.0"
+}
+```
+
 ## 3.14 `tiles`
 ## 3.15 `vector_layers`
 ## 3.16 `version`
```

Comment:
> "Implementations MUST NOT use tiles with different major versions." what does this actually mean? Can we provide an example to make it more clear?

Same. Perhaps it's saying, any implementation must ensure its tilejson major version is consistent. If updating tilejson major version, it must apply to all uses of tilejson throughout the system.

...?

Reply:
> updated the first sentence to include "... representing the vector tiles themselves." to be more explicit. Thoughts?

From what I understand, the `version` field here is referring to the `tilejson` version, not the `vector-tile` version.

Reply:
Oh pardon, the `tilejson` field is referring to the tilejson version.

The `version` field is referring to the vector-tile version. Therefore, your addition sounds great. I think also adding something like:
> A semver.org style version number of the tiles

Also, can we assume it's always vector-tiles?

Reply:
> Also, can we assume it's always vector-tiles?

Good point, I don't think we can. In fact now that I re-read my added example text "such as polygon or linestring geometry updates" I'm wondering if this isn't actually true, and is more related to breaking changes of the underlying data. At least for vector tiles, this would be something like "changing the name of `layerA` to `layer1`" which would break downstream tools referencing the data. I'm not exactly sure how you'd version raster data though.

Reply:
Updated the "major" example to layer name changes, and just say "tiles" instead of "vector tiles" since these could be raster tiles.